lib/Service/RoomService.php

```diff
@@ -362,6 +366,54 @@ public function setAvatar(Room $room, string $avatar): bool {
 		return true;
 	}
 
+	public function startRecording(Room $room, $status): bool {
+		$availableRecordingTypes = [Room::RECORDING_VIDEO, Room::RECORDING_AUDIO];
+		if (!in_array($status, $availableRecordingTypes)) {
+			throw new InvalidArgumentException('status');
+		}
+		if ($room->getCallRecording() !== 0) {
+			throw new InvalidArgumentException('room');
+		}
+		return $this->setCallRecording($room, $status);
+	}
+
+	public function stopRecording(Room $room): bool {
+		if ($room->getCallRecording() === 0) {
```

Comment:
```suggestion
		if ($room->getCallRecording() === Room::RECORDING_STOP) {
```
or if you follow my suggestion to rename it to "none":
```suggestion
		if ($room->getCallRecording() === Room::RECORDING_NONE) {
```